pymesync/tests.py

```diff
@@ -2369,6 +2369,11 @@ def test_project_users_no_project_parameter(self):
                           {self.ts.error: "Missing project slug, please "
                                           "include in method call"})
 
+    def test_baseurl_with_trailing_slash(self):
```

Comment:
A more thorough test would also set a url without trailing slash and make sure the value is the same as setting with the slash.